### DISCUSSION TITLE
Publishers - Group with additional set of roles and permissions

### DISCUSSION THREAD

**Discussion by stefan-korn** [GetDKAN/dkan#4118] [Q&A]
The demo website states on the publishers page

> Groups allow you to classify datasets that share a common publisher or organizational group (i.e. Parks and Recreation Department, Department of Education). Behind the scenes, Groups can offer an additional set of roles and permissions that ensure quality and security when publishing data. This is especially important for large sites that may have several working groups publishing data to the same site.

I could not find an explanation how to use this feature (additional set of roles and permission) and where to find the set of roles and permissions regardings Groups.

Is this to be found on the Drupal part or in the React App?

**Comment by janette** [comment]
@stefan-korn I've updated the text to make it clear that you need to add the Drupal/group module to your build for this feature. That update should be visible after the next DKAN release.

**Comment by stefan-korn** [comment]
Coming back to this once more. Am I right that there is curently no DKAN specific integration of  the Group module? No relation between schema properties and Groups, so for example to map a publisher from schema to a Group?

**Comment by dafeder** [comment]
@stefan-korn at the moment, no. Would be open to suggestions on how to implement this.
